docs/content/v0.20/contributor.md

```diff
@@ -0,0 +1,60 @@
+# KubeStellar Contributors
+
+**NOTE**: work in progress... write more
+
+## Check key pre-requisites for KubeStellar
+
+The [check_pre_req](../../../hack/check_pre_req.sh) script offers a convenient way to check key pre-requisites needed for KubeStellar deployment and use case scenarios.
+
+The [check_pre_req](../../../hack/check_pre_req.sh) script check for a pre-requisite presence in the path, by using the `which` command, and it can optionally provide version and path information for pre-requisites that are present, or installation information for missing pre-requisites.
+
+We envision that the [check_pre_req](../../../hack/check_pre_req.sh) script could be useful for user-side debugging as well as for asserting the presence of pre-requisites in higher-level automation scripts.
+
+The [check_pre_req](../../../hack/check_pre_req.sh) script accepts a list of optional flags and arguments.
+
+**Supported flags:**
+
+- `-A|--assert`: exits with error code 2 upon finding the fist missing pre-requisite
+- `-L|--list`: prints a list of supported pre-requisites
+- `-V|--verbose`: displays version and path information for installed pre-requisites or installation information for missing pre-requisites
+- `-X`: enable `set -x` for debugging the script
+
+**Supported arguments:**
+
+The [check_pre_req](../../../hack/check_pre_req.sh) script accepts a list of specific pre-requisites to check, among the list of available ones:
+
+```shell
+$ check_pre_req.sh --list
+argo brew docker go helm jq kcp kflex kind ko make ocm yq
+```
+
+For example, the complete list of pre-requisites could be checked with the command below:
+
+```shell
+$ check_pre_req.sh
+X ArgoCD CLI
+✔ Home Brew
+✔ Docker
+✔ Go
+✔ Helm
+✔ jq
+X kcp

```

Comment:
I wouldn't put KCP as pre-req for the default (i.e. no parameters) of this script. Obviously one can still explicitly check for it using this scrip